homeassistant/components/device_tracker/unifi.py

```diff
@@ -41,6 +43,8 @@
         cv.boolean, cv.isfile),
     vol.Optional(CONF_DETECTION_TIME, default=DEFAULT_DETECTION_TIME): vol.All(
         cv.time_period, cv.positive_timedelta),
+    vol.Optional(CONF_EXTRA_ATTRIBUTES, default=DEFAULT_EXTRA_ATTRIBUTES):
```

Comment:
`default=False` is better then `default=DEFAULT_EXTRA_ATTRIBUTES`

Reply:
It would not be consistent we the remaining DEFAUT_*